include/elf_parser.hpp

```diff
@@ -0,0 +1,191 @@
+#pragma once
+#include <bitset>
+#include <err.h>
+#include <fcntl.h>
+#include <gelf.h>
+#include <libelf.h>
+#include <print>
+#include <stdlib.h>
+#include <string>
+#include <sysexits.h>
+#include <unistd.h>
+
+class ElfParser
+{
+  public:
+    ElfParser(char** t_file_ptr)
+      : m_file(t_file_ptr){};
```

Comment:
```suggestion
    ElfParser(char** p_file_ptr)
      : m_file(p_file_ptr){};
```

Input parameters should have prefix `p_`